sgfdevs-frontend/src/app/about/code-of-conduct/page.tsx

```diff
@@ -0,0 +1,13 @@
lack thereof), general lifestyle choices, dietary preferences, or technology choices.</p><p><strong>Applicability.</strong> This Code of Conduct applies to all Method Conference organizers, volunteers, sponsors, speakers, and participants. Vendors will also be held to the same standards. We reserve the right to remove any person who does not adhere to the Code of Conduct.</p><p><strong>Untolerated Behaviors.</strong> We do not tolerate harassment of any kind, including but not limited to the following behaviors, especially regarding the above-mentioned Method values:</p><ul><li>Intimidation or threats</li><li>Inappropriate disruption of events</li><li>Hurtful language</li><li>Physical assault of any kind</li><li>Inappropriate physical contact</li><li>Unwanted sexual attention</li><li>Sexual language or imagery</li><li>Unwanted photography or video recording</li><li>Bullying or stalking</li><li>Possession of any kind of weapon</li></ul><p><strong>Reporting Behaviors Inconsistent With Code of Conduct.</strong> Method Conference is a “see-something, say-something” community. If you witness or are subject to any behaviors listed above or those that appear inconsistent with the stated Method values, you can make a report in one of the following ways:</p><ul><li>Submit an anonymous report through the Method Conference Mobile App (which will notify designated conference organizers/volunteers immediately)</li><li>Text Shawna Baron at 417-894-9926</li><li>Locate a conference organizer or volunteer, who are recognizable by their badges/t-shirts (or as otherwise explained by organizers at the beginning of the conference). In an emergency, please call 911.</li></ul><p><strong>Response By Method Conference Staff.</strong> It is the goal of Method Conference organizers and volunteers to resolve the reported issue as quickly as possible as to ensure the safety of all. If we cannot immediately resolve the issue, we will do our best to set proper expectations of when we will be able to. We will take any action we deem appropriate, up to and including removal from the conference premises. At minimum, anyone asked to stop engaging in a potentially unsafe behavior is expected to comply immediately. Conference organizers and volunteers can also help you in contacting law enforcement, providing escorts, or otherwise assist those experiencing harassment to feel safe for the duration for the conference. If the person being reported is a conference organizer or volunteer, they will be expected to recuse themselves immediately from handling your incident.</p><p><strong>Speaker content.</strong> Speakers at Method Conference must agree to this Code of Conduct before they can be listed as confirmed speakers on the website. They must also agree their content is educational in nature and adheres to the spirit of the Code of Conduct (e.g., no potentially offensive materials such as sexual language or imagery, gratuitous profanity, etc).</p></div> </div></div>  </body></html>';
```

Comment:
For the example could you strip out all the tags that wouldn't show up for a typical rich text editor? e.g. `DOCTYPE`, `html`, `body`, `div` etc

Reply:
Done!